tests/Hedgehog.CSharp.Tests/LinqTests.cs

```diff
@@ -1,7 +1,8 @@
 ﻿using Xunit;
 
-// Import Check and ForAll:
-using static Hedgehog.Property;
+// Import ForAll:
+using Hedgehog.CSharp;
```

Comment:
```suggestion

```

I don't think we need to have this using directive anymore. The other below also exposes `ForAll`.